src/main/java/org/broadinstitute/hellbender/utils/gcs/BucketUtils.java

```diff
@@ -54,6 +54,10 @@ public static boolean isCloudStorageUrl(final String path) {
         return path.startsWith(GCS_PREFIX);
     }
 
+    public static boolean isCloudStorageUrl(final java.nio.file.Path path) {
+        return (path.toUri().getScheme() + "://").equals(GCS_PREFIX);
```

Comment:
Seems like `getScheme()` can return null, so we should guard against that possibility.